web/public/js/controllers/manage-event-applications-controller.js

```diff
@@ -86,11 +86,22 @@
 
     event.listDownloadLink = function (status) {
       cdEventsService.exportGuestList(dojoId, event.id, status, function (response) {
+        //  TODO: export as a directive, it modifies the DOM
+        //  param : name, datasource call (cdEvents.xxx)
         var downloadLink = angular.element('<a></a>');
-        var csv = new Blob([response], { type: "text/csv;charset=utf-8;" });
-
-        downloadLink.attr('href',(window.URL || window.webkitURL).createObjectURL(csv));
-        window.open(downloadLink[0]);
+        var csv = new Blob([response.data], { type: "text/csv;charset=utf-8;" });
+        var urlFactory = window.URL || window.webkitURL;
+        var tempFile = urlFactory.createObjectURL(csv);
+        downloadLink.attr('href', tempFile);
+        downloadLink.attr('download', $scope.event.name + '_' + $translate.instant('Applicants') + moment().format() + '.csv');
```

Comment:
Actually, the dojo name is even more dangerous ;)
Tho, it seems createObjectURL does URI encode by default, so that it's still usable. We could encoreURI the translation just to be sure ?